### PR TITLE
Show secrets detected at PR scan

### PR DESCRIPTION
**Pull request by attiasas**
- [ ] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
- [ ] Update [documentation](https://github.com/jfrog/documentation) about new features / new supported technologies
---

Add an option to generate secrets comments in scan PR using `JF_PR_SHOW_SECRETS_COMMENTS=true`
image

### CODE REVIEW

File: utils/outputwriter/outputcontent.go [hunk 1]
```diff
@@ -335,7 +338,30 @@ func ApplicableCveReviewContent(severity, finding, fullDetails, cve, cveDetails,
 }
 
 func getJasDescriptionTable(severity, finding string, writer OutputWriter) string {
-	return NewMarkdownTable("Severity", "Finding").AddRow(writer.FormattedSeverity(severity, "Applicable"), finding).Build()
+	return NewMarkdownTable("Severity", "Finding").AddRow(writer.FormattedSeverity(severity, jasutils.Applicable.String()), finding).Build()
+}
+
+func getSecretsDescriptionTable(severity, finding, status string, writer OutputWriter) string {
+	columns := []string{"Severity", "Finding"}
+	applicability := jasutils.Applicable.String()
```
Comment (hadarshjfrog) [inline utils/outputwriter/outputcontent.go:346]: We don't have `undetermined` for Secrets' Applicability? only `Applicable ` and `NotApplicable `?

File: testdata/messages/reviewcomment/secrets/secret_review_content_no_ca_simplified.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+
+
+---
+## 🗝️ Secret Detected
+
+---
+| Severity                | Finding                  |
+| :---------------------: | :-----------------------------------: |
+| Medium | Secret keys were found |
```
Comment (hadarshjfrog) [inline testdata/messages/reviewcomment/secrets/secret_review_content_no_ca_simplified.md:9]: The new default for secrets in now `High` and not `Medium` - please change the tests accordingly

Comment (attiasas) [reply testdata/messages/reviewcomment/secrets/secret_review_content_no_ca_simplified.md:9]: Its just a snapshot and example...

File: utils/outputwriter/outputcontent.go (hunk 1 above)
Comment (attiasas) [reply utils/outputwriter/outputcontent.go:346]: we have a diff statuses for it, I use the Applicable for the image that will be shown.
If `Inactive` I will show the severity image like `Not Applicable`, all other statuses are shown with severity image of `applicable`

File: utils/comment.go [hunk 3]
```diff
@@ -201,6 +202,12 @@ func getNewReviewComments(repo *Repository, issues *IssuesCollection) (commentsT
 	for _, sast := range issues.Sast {
 		commentsToAdd = append(commentsToAdd, generateReviewComment(SastComment, sast.Location, generateSourceCodeReviewContent(SastComment, sast, writer)))
 	}
+	if !repo.Params.PullRequestSecretComments {
+		return
+	}
+	for _, secret := range issues.Secrets {
```
Comment (eranturgeman) [inline utils/comment.go:208]: I saw you changed in one of your recent PRs to iterate over unique issues (iac, sast - GetUniqueIacIssues, GetUniqueSastIssues)
Should we do the same here?

**Review by eranturgeman — APPROVED** [review]

File: utils/comment.go (hunk 3 above)
Comment (attiasas) [reply utils/comment.go:208]: violation not exists here

File: utils/outputwriter/outputcontent.go [hunk 4]
```diff
@@ -19,8 +20,10 @@ const (
 	vulnerableDependenciesResearchDetailsSubTitle = "🔬 Research Details"
 
 	contextualAnalysisTitle = "📦🔍 Contextual Analysis CVE Vulnerability"
-	iacTitle                = "🛠️ Infrastructure as Code Vulnerability"
-	sastTitle               = "🎯 Static Application Security Testing (SAST) Vulnerability"
+	//#nosec G101 -- not a secret
+	secretsTitle = "🗝️ Secret Detected"
```
[the remark this answers is not in the document]
Comment (attiasas) [reply utils/outputwriter/outputcontent.go:24]: done in the next PR with violations

**Pull request merged**